src/mui-omni-search.js

```diff
@@ -12,7 +12,7 @@ import { searchIndexLevelMap } from './constants/campaing-level'
 
 
 const useStyles = makeStyles(() => ({
-  container: { width: '30rem' },
+  container: { margin: '5px' },
```

Comment:
could potentially take advantage of `theme`

```
const useStyles = makeStyles((theme) => ({
  container: { margin: theme.spacing(0.2) },
}))
```

which is relative to the parent theme config (falls back to MUI default theme) which usually gives a more consistent feeling than pixel-level definition. this is also applicable for `padding` definition.